goldrush_path/opt.cpp

```diff
@@ -66,7 +66,7 @@ print_usage(const std::string& progname)
        "  -p prefix               write output to files with prefix "
        "[goldrush_out]\n"
        "  -P PHRED_AVG            minimum average phred score for each read "
-       "[15]\n"
+       "[0 (calculates phred score minimum automatically)]\n"
```

Comment:
Suggestion to make this consistent with the suggestions I had in the help pages above.